File: python/ex1.py
def main():
    counter = 0
    st = input("enter new string: ")
    while len(st) > 4:
        if "T" in st:
            counter += 1
        st = input("enter new string: ")
    print(f"T appears {counter} times")


if __name__ == '__main__':
    main()

File: Java/src/Ex1.java
import java.util.Scanner;

public class Ex1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String str;
        int counter = 0;
        System.out.println("enter new string: ");
        str = in.nextLine();
        while (str.length() > 4) {
            if (str.contains("T")) {
                counter++;
            }
            System.out.println("enter new string: ");
            str = in.nextLine();
        }

        System.out.printf("T appears %d times.", counter);
        in.close();
    }
}
